### PR TITLE
Move type definitions out of settings

### LINKED ISSUE #469: Move type definitions out of settings.ts
We need to move all the type definitions out of settings.ts and into something like types.ts.  This is based on the conversation here https://github.com/onaio/reveal-frontend/pull/459#discussion_r329840979

### PR DESCRIPTION
**Pull request by DavisRayM**
Moves type definitions out of settings.ts into a separate types.ts file adjacent to helpers.ts.

Fix #469

### CODE REVIEW

File: src/configs/settings.ts [hunk 1]
```diff
@@ -13,6 +13,13 @@
  */
 import { Providers } from '@onaio/gatekeeper';
 import { Expression, LngLatBoundsLike } from 'mapbox-gl';
+import {
+  ActionReasonType,
+  GoalPriorityType,
+  PlanActionCodesType,
+  PlanActivities,
+  UseContextCodesType,
+} from '../containers/forms/PlanForm/types';
```
Comment (cKellyDesign) [inline src/configs/settings.ts]: Seems like this is a little circular, but for this type of thing (pun intended) I think it makes sense!

File: src/configs/types.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import {
+  adminLayerColors,
+  GREEN_THRESHOLD,
+  IndicatorRowItem,
+  JurisdictionLevels,
+  ORANGE_THRESHOLD,
+  YELLOW_THRESHOLD,
+} from './settings';
```
Comment (cKellyDesign) [inline src/configs/types.ts]: Wonderful

**Comment by moshthepitt** [comment]
@DavisRayM please update the note at the top of the settings module and then we can merge

File: src/configs/settings.ts (hunk 1 above)
Comment (moshthepitt) [reply src/configs/settings.ts]: yeah, lol, :)

**Pull request merged**